keep/api/models/db/preset.py

```diff
@@ -1,13 +1,18 @@
 from uuid import UUID, uuid4
-
+from typing import Optional
 from pydantic import BaseModel
 from sqlmodel import JSON, Column, Field, SQLModel
 
 
 class Preset(SQLModel, table=True):
     # Unique ID for each preset
     id: UUID = Field(default_factory=uuid4, primary_key=True)
+
     tenant_id: str = Field(foreign_key="tenant.id", index=True)
+
+    # if created_by is not null then it means this preset is only for this user
+    # keeping index=True for better search
+    created_by: Optional[str] = Field(index=True, nullable=True)
```

Comment:
I'd add both `created_by` and `is_private` so `created_by` says who created this Preset (this is always someone) and `is_private` could hold the logic to whether this is owned by specific person (not null) or public (null), wdyt?

Reply:
Yep should be this way from the start, will be done!